impeller/toolkit/interop/typography_context.h

```diff
@@ -29,8 +30,22 @@ class TypographyContext final
 
   const std::shared_ptr<txt::FontCollection>& GetFontCollection() const;
 
+  //----------------------------------------------------------------------------
+  /// @brief      Registers custom font data. If an alias for the family name is
+  ///             provided, subsequent lookups will need to use that same alias.
+  ///             If not, the family name will be read from the font data.
+  ///
+  /// @param[in]  font_data          The font data
+  /// @param[in]  family_name_alias  The family name alias
+  ///
+  /// @return     If the font data could be successfully registered.
+  ///
+  bool RegisterFont(std::unique_ptr<fml::Mapping> font_data,
```

Comment:
This should take a string_view since we're just going to immediately copy it with an SkString